hdf5-vol-daos.spec

```diff
@@ -58,7 +58,7 @@ Patch1: https://github.com/HDFGroup/vol-daos/commit/5eed612db03458c1b21e18f13fa2
 Patch2: https://github.com/HDFGroup/vol-tests/commit/e29abaf69233e014e6e26cf927cc55934c178128.patch
 Patch3: https://github.com/HDFGroup/vol-tests/commit/8a4d3079ebf7ddb4222183f03be6e7bd33a86ee2.patch
 Patch4: https://github.com/HDFGroup/vol-daos/commit/88cefd445aa8cb7bc33f9bf30355501138866e27.patch
-Patch5: https://github.com/HDFGroup/vol-daos/commit/a250f01c40d2f845b01ea50531227f35a44a4af3.patch
+Patch5: a250f01c40d2f845b01ea50531227f35a44a4af3.patch
```

Comment:
This actually is the preferred (at least traditional) format.  Patches being local files that is.  I suppose with SCMs being more online these days, migration toward URLs for patches is not unnatural.